CONTRIBUTING.md

```diff
@@ -11,26 +28,118 @@
+- `Step 3`: Create your pack!
+- `Step 4`: Submit a PR.
+- `Step 5`: resolve feedback from code reviews
+- `Step 6`: after approval, merge your PR
+
+When creating a new Observability Pack or reviewing a PR please keep the following in mind, and refer to the
+[Packs validation workflow](https://github.com/newrelic/newrelic-observability-packs/blob/main/.github/workflows/validate_packs.yml) for up to date validations.
+
+### Required field best practices
+
+#### Name
+
+> The official name of the pack.
+
+- Required
+- limited to 100 characters
+- The `Validate Packs / Ensure pack names are unique` job verifies name uniqueness.
+- Avoid PR collisions by checking if any [open PRs](https://github.com/newrelic/newrelic-observability-packs/pulls) are using the same name you wish to use.
+- Pack names are stripped of any punctuation and white space is replaced by `-` before doing any comparisons.
+
+#### Description
+
+> A detailed description of the pack and why it's useful.
+
+- Required
+- Plain text only
+- limited to 2000 characters
+- The `TBD` job verifies pack descriptions
+
+#### Level
+
+> The support level provided for the pack.
+
+- Required
+- set to `Community` level by default
+- Can only be modified by New Relic employees to set another level type
+- The `TBD` job verifies pack level
+
+#### Author
+
+> The creator(s) of the pack
+
+- Required
+- No limit to the amount of authors
+- The `TBD` job verifies pack level
+
+### Optional field best practices
+
+Other fields in a pack are optional but it's recommended you consider the following to offer the best possible pack
+experience for users.
+
+#### Short Description
+
+> A brief summary of the pack functionality.
+
+- Optional
+- Plain text only
+- limited to 250 characters
+
+### Images or screenshots
+
+> Images of dashboards, visualizations, or nerdpacks.
+
+- Optional
+- jpeg or png format
+- 4MB or smaller
+- 6 max images
+
+#### Image recommended for best results
+
+- Aspect ratio: 3:2
+- 800 px (width)
+- 1600 px (height)
+
+### Logos
+
+> The logo or brand image of the pack.
+
+- Optional
+- .png or .jpeg or .svg format
+- Max 1
+
+#### logo recommendations or best results
+
+- Aspect ratio: 1:1
+- 250px (width) x 100px (height)
 
-> Review the [pack template config](./_template/config.yml) file for a definition of how to create a pack.
+### Website URLs
 
-If you decide to create a new Observability Pack or review a PR please keep the following in mind.
+> a reference to a website relating tot he pack.
 
-* Your new pack must have a unique name
-  * The `Validate Packs / Ensure pack names are unique` job handles verifying that there are no collisions when you create a pull request
-  * **Note:** Names are stripped of any punctuation and white space is replaced by `-` before doing any comparisons
+- Optional
+- Valid URL only https://www.newrelic.com
```

Comment:
My $0.02 - this should be a link to the `main_config.md` doc rather than being duplicated info that could go stale as the doc gets updates. I think the `docs` directory should be our source of truth as much as possible. 

It might be worth copying some of the info that's here into `main_config.md`.

Reply:
that's a good point.. I wasn't sure how much you wanted to make the `config-spec.md` a how to guide. So the intention here was to step a user through what to think about when creating a pack. I'm happy to transfer a lot of this content to that file if that is where you'd prefer to see it.

Reply:
I think much of this is already encapsulated in the spec file (required/optional, limits, format, etc.). Main concern is if it's represented here, it's just one more thing to be mindful of if the spec is modified.

Reply:
But if you find value in keeping it here for contributors, that's cool too

Reply:
👍